From 5fcfc1acd2620c786fb1b3b741d2a7cbee4445cb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=B6rg=20Sonnenberger?= <joerg@NetBSD.org>
Date: Wed, 26 Nov 2014 14:33:38 +0100
Subject: [PATCH] Remove unnecessary check.

The vmods member of XkbNamesPtr is an array and thus never NULL.

Signed-off-by: Thomas Klausner <wiz@NetBSD.org>
Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
(cherry picked from commit 9fa04388dcbb5bf205319569f540a62c03754688)
---
 src/cout.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/cout.c b/src/cout.c
index 399512f..ee0971e 100644
--- a/src/cout.c
+++ b/src/cout.c
@@ -45,7 +45,7 @@ WriteCHdrVMods(FILE *file,Display *dpy,XkbDescPtr xkb)
 {
 register int i,nOut;
 
-    if ((!xkb)||(!xkb->names)||(!xkb->names->vmods))
+    if ((!xkb)||(!xkb->names))
 	return False;
     for (i=nOut=0;i<XkbNumVirtualMods;i++) {
 	if (xkb->names->vmods[i]!=None) {
-- 
2.3.5

